mappings/net/minecraft/block/CarvedPumpkinBlock.mapping

```diff
@@ -1,5 +1,19 @@
 CLASS bmy net/minecraft/block/CarvedPumpkinBlock
 	FIELD a FACING Lbwe;
+	FIELD b headlessSnowmanPattern Lbvo;
+	FIELD c snowmanPattern Lbvo;
```

Comment:
snowGolemXX, as it's going to be renamed by #657

Reply:
`headlessSnowman`...?

Reply:
The `headlessXxx` patterns are the same as the regular patterns but are missing the pumpkin on top, so, headless. Dispensers probably use these when dispensing a pumpkin but I haven't checked.

Reply:
Actually duhhhh, the Wither skull already has these mapped so I should copy those